src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -392,9 +395,10 @@ public Task StartHostAsync(RHostStartupInfo startupInfo, IRSessionCallback callb
         private async Task AfterHostStarted(RHostStartupInfo startupInfo) {
             var evaluator = new BeforeInitializedRExpressionEvaluator(this);
             try {
+                var rHostAssembly = _services.GetService<ILocalClientServices>().GetAssemblyByType(typeof(RHost));
                 // Load RTVS R package before doing anything in R since the calls
                 // below calls may depend on functions exposed from the RTVS package
-                var libPath = IsRemote ? "." : Path.GetDirectoryName(Assembly.GetExecutingAssembly().GetAssemblyPath());
+                var libPath = IsRemote ? "." : Path.GetDirectoryName(rHostAssembly.GetAssemblyPath());
```

Comment:
fixed

---
In reply to: [111820628](https://github.com/Microsoft/RTVS/pull/3419#discussion_r111820628) [](ancestors = 111820628)